src/index.html

```diff
@@ -10,23 +10,49 @@
       http-equiv="X-UA-Compatible"
       content="ie=edge"
     />
-    <title>Document</title>
+    <title>Mate academy | airbnb search bar</title>
     <link
       rel="stylesheet"
       href="style.css"
     />
-  </head>
-  <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
+    <link
+      rel="stylesheet"
+      href="search-bar.css"
     />
+  </head>
+  <body class="page">
+    <form
+      class="form page__form form--big"
+      action="GET"
+      data-qa="big"
+    >
+      <label
+        class="label label--big"
+        data-qa="big-bar"
+      >
+        <input
+          class="label__input input--big"
```

Comment:
You have used modifier (input--big) on input without using block (input) className. Its one of BEM mistakes.